filtered_registries/dgen_multiple_metrics/dsgrid-project-IEF/project/dimensions/enduses_tmp.csv

```diff
@@ -0,0 +1,43 @@
+id,name,fuel_id,unit
+
+electricity_computers_and_tvs,TVs and computers,electricity,MWh
+electricity_drying,drying,electricity,MWh
+electricity_cooking,cooking,electricity,MWh
+electricity_cooling_and_heating,HVAC,electricity,MWh
+electricity_fans_and_pumps,fans and pumps,electricity,MWh
+electricity_lighting,lighting,electricity,MWh
+electricity_miscellaneous_electrical_loads,MELs/other,electricity,MWh
+electricity_non_pc_office_equipment,Non-PC Office Equipment,electricity,MWh
+electricity_other,other,electricity,MWh
+electricity_refrigeration,refrigeration,electricity,MWh
+electricity_water_heating,water heating,electricity,MWh
+electricity_pools_and_spas,pools and spas,electricity,MWh
+electricity_dishwasher,dishwasher,electricity,MWh
+
+
+
```

Comment:
Are these blank lines (and above) problematic? Why is this `enduses_tmp.csv`?

Reply:
Will delete...not sure where this came from.